.github/workflows/ci.yml

```diff
@@ -75,25 +75,35 @@ jobs:
   unit-tests:
     name: Unit tests
     needs: [pre-commit, towncrier, package]
+    runs-on: ${{ matrix.platform }}-latest
+    continue-on-error: ${{ matrix.experimental }}
     strategy:
       matrix:
         platform: ["macos", "ubuntu", "windows"]
         python-version: ["3.8", "3.9", "3.10", "3.11", "3.12-dev"]
+        use-win-store-python: [ false ]
         include:
-          - experimental: false
-
-          - python-version: "3.12-dev"
-            experimental: true
-    runs-on: ${{ matrix.platform }}-latest
-    continue-on-error: ${{ matrix.experimental }}
+        - experimental: false
+        - python-version: "3.12-dev"
+          experimental: true
+        - platform: "windows"
+          python-version: "3.11"
+          use-win-store-python: true
+          experimental: false
     steps:
     - uses: actions/checkout@v3.3.0
       with:
         fetch-depth: 0
     - name: Set up Python
+      if: matrix.use-win-store-python == false
       uses: actions/setup-python@v4.5.0
       with:
         python-version: ${{ matrix.python-version }}
+    - name: Install Windows Store Python
+      if: matrix.use-win-store-python == true
+      uses: ./.github/actions/install-win-store-python
```

Comment:
Presumably this would be updated to point at the `.github` repo version when beeware/.github#5 lands?

Reply:
Correct.

Reply:
Updated now.